mod_wsbridge/mod_wsbridge.c

```diff
@@ -700,6 +789,30 @@ wsbridge_callback_ws(struct lws *wsi, enum lws_callback_reasons reason,
 			tech_pvt->has_dtmf = FALSE;
 		}
 		switch_mutex_unlock(tech_pvt->dtmf_mutex);
+
+		{
+			char* event_message = NULL;
+			void* pop;
+			if (Queue_pop(&tech_pvt->eventQueue, &pop) == SWITCH_STATUS_SUCCESS) {
```

Comment:
We don't need `event_message`. Just cast `pop` to a `char*` when calling cJSON_Parse
`json_message = cJSON_Parse((char *) pop);`